### DISCUSSION TITLE
Did you experience your computer never going to sleep after installing this program?

### DISCUSSION THREAD

**Discussion by KaijiGit** [DistroAV/DistroAV#1031] [Q&A - Support]
Hi guys,

This program is detected as a spy tool on virus total by several virus detection companies so I hesitated a long time before installing it and I finally decided to install it thinking it was a false positive but since I did my computer is now never going to sleep and you all know how hard it is to prevent a windows system from going to sleep, do you experience the same thing on your side after installing this tool ? 

Thank you 

Kaiji

**Comment by paulpv** [comment]
OBS by design keeps your PC awake while it is capturing.
This is not due to OBS-NDI.
Try yourself by uninstalling OBS-NDI but leave OBS running.
Your PC will continue to stay awake even without OBS-NDI installed (or having never been installed).

OBS-NDI is open source and nothing in its or dependents code is a virus.
You could take our word for it or check out the code yourself.

Yes, a few AV apps do mistake the **installer** as a virus; my best guess is probably because lots of viruses do use https://jrsoftware.org/isinfo.php to package themselves.
